trees/README.md

```diff
@@ -5,3 +5,7 @@
 1. [Pre, In & Post Order Traversals](c-or-cpp/pre-in-post-traversal.cpp)
 2. [Level Order Traversal](c-or-cpp/level-order-traversal.cpp)
 3. [Build Binary Tree using Pre,In & Post Order](c-or-cpp/build-binary-tree.cpp)
+
+### Java
+
+1. [Pre, In & Post Order Traversals](java/pre_in_post_traversal.java)
```

Comment:
Hi @toihirhalim 

The file's and folder's name should be in lowercase, and if has whitespace should be separate by `-`

Reply:
thanks @TawfikYasser  for the review 
so the java file name: white spaces are separated by `_`
when I try using `-` it fails to compile

Reply:
Don't worry about it

Reply:
Alright